libnavui-dropin/src/test/java/com/mapbox/navigation/dropin/component/routefetch/RoutesViewModelTest.kt

```diff
@@ -48,15 +47,18 @@ internal class RoutesViewModelTest {
     @Before
     fun setUp() {
         mockkStatic("com.mapbox.navigation.base.internal.extensions.ContextEx")
-        routesStateFlow = MutableStateFlow(RoutesState.INITIAL_STATE)
         navigationStateFlow = MutableStateFlow(NavigationState.FreeDrive)
         navigationStateViewModel = mockk {
             every { state } returns navigationStateFlow
         }
         locationViewModel = mockk(relaxed = true) {
             every { lastPoint } returns currentLoc
         }
-        sut = RoutesViewModel(navigationStateViewModel, locationViewModel, routesStateFlow)
+        sut = RoutesViewModel(
```

Comment:
NIT: Can we rename this variable?

Reply:
sut == SystemUnderTest 

cc: will let that discussion follow with @tomaszrybakiewicz :). When updating, will need to update a few tests

Reply:
https://github.com/ghughal/software-development-ebooks/blob/master/%5BxUnit%20Test%20Patterns%20Refactoring%20Test%20Code%20(Addison-Wesley%20Signature%20Series%20(Fowler))%20Kindle%20Edition%20by%20Gerard%20Meszaros%20-%202007%5D.pdf

Asked tomasz about it when i noticed and he referenced that book. There's a free download of the book where the author is high on the SUT concept 😄